doc/rst/language/evolution.rst

```diff
@@ -3,18 +3,225 @@
 Chapel Evolution
 ================
 
-Like any language, Chapel has changed over time. This page is designed to
-capture significant language changes that have the possibility of breaking
-existing user codes or code samples from old presentations or papers that
-predated the changes.
+Like any language, Chapel has changed over time. This page is designed
+to describe significant language changes that have a high likelihood
+of breaking existing user codes or code samples from presentations or
+papers that predated the changes.
 
 Note that the compiler flag ``--warn-unstable`` is available and can be
 useful when migrating programs to the current version of the language.
-The purpose of this flag is to identify  portions of a program that use a
-language feature that has changed meaning.
+The purpose of this flag is to identify portions of a program that use a
```

Comment:
Has this flag's meaning changed overtime? I have always thought of it as the latter description (considered unstable and can change in the future).

Reply:
I would say it has evolved over time.  Historically, we've used it primarily to warn people about things that had already changed, such as `new C()` no longer meaning `new unmanaged C()` (and then no longer meaning `new borrowed C()`).  In this release we added a lot more things in the "be forewarned / may be likely to change" category to it, which is why I updated the wording here.